std_msgs/Header header
int16[] audio        # 오디오 (16-bit PCM 데이터)
std_msgs/Int32 doa   # 음원 방향 (도 또는 radian 등 정의에 따름)
